metadata.rb

```diff
@@ -4,14 +4,14 @@
 license          'Apache-2.0'
 description      'Installs/Configures Splunk Servers and Forwarders'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.14.0'
```

Comment:
Typically I defer this change until the merge, but *shrug*

Reply:
I had to do it or berks would fail to resolve the dependencies. Since the previous version is available in the supermarket and I changed the version of chef-vault, it kept telling me I had conflicting dependencies.

Reply:
Interesting. I wonder if there was a change to Berks, I thought with the metadata directive it was supposed to favor the local metadata.rb.  Although I could see a rogue lock file doing things too... But as I mentioned before *shrug* :)

Reply:
Interesting. I wonder if there was a change to Berks, I thought with the metadata directive it was supposed to favor the local metadata.rb.  Although I could see a rogue lock file doing things too... But as I mentioned before *shrug* :)